README.md

```diff
@@ -32,6 +32,40 @@ and then:
 
 When running with `parallel` set to true, the final reports can be merged by using `ember coverage-merge`. The final merged output will be stored in the `coverageFolder`.
 
+## TypeScript integration
+
+Steps:
+
+* in `tsconfig.json`
+```js
+  {
+    "compilerOptions": {
+      "inlineSourceMap": true,
+      "inlineSources": true
+    }
+  }
+```
+* in `ember-cli-build.js`
+```js
+  const app = new EmberApp(defaults, {
+    babel: {
+      sourceMaps: 'inline'
+    },
+    sourcemaps: {
+      enabled: true,
+      extensions: ['js']
+    }
+  });
+```
+* in `package.json` specify latest available version
+```js
+  {
+    devDependencies: {
+      "ember-cli-code-coverage": "https://github.com/kategengler/ember-cli-code-coverage#master#2018-08-15T15:51:29Zs"
```

Comment:
It's not necessary to pin to master. It should work fine with the latest beta.

Reply:
@jamescdavis last published to npm version is - `0.4.2`. (https://www.npmjs.com/package/ember-cli-code-coverage)